SwiftReflector/OverrideBuilder.cs

```diff
@@ -719,10 +724,16 @@ SLSubscript DefineOverrideSubscript (FunctionDeclaration getter, FunctionDeclara
 				                                                                  getParams.Select (nameTypePair => nameTypePair.PrivateName)), new SLIdentifier ("newValue")));
 				elseblock.Add (superAssign);
 				var ifelse = new SLIfElse (condition, ifblock, elseblock);
-				if (isProtocol)
-					setBlock = ifblock;
-				else
+				if (isProtocol) {
```

Comment:
I've seen this "if 0 set block else add range" pattern before. Worth an extension method?

Reply:
This code is likely to go away entirely about 2 PRs in the future, so I'm not concerned.